Resources/Help/analyses/ancova.md

```diff
@@ -70,24 +70,25 @@ ANCOVA allows the user to analyze the difference between multiple group means, w
 ### Post Hoc Tests
 - To perform a post hoc test, drag one or more factor names to the right column. Several options are available:    
 - Type: Different types of post hoc tests can be selected. 
-      -  Standard: Pairwise t-tests are performed. All the corrections can be applied to this method. This option is selected by default. 
-        - Confidence intervals: When this option is selected, the confidence interval for the mean difference is calculated. This is done for every post hoc method except for Dunn. By default this is set to 95% but this can be adjusted into the desired percentage.  
-        - From `...` bootstraps: By selecting this option, the bootstrapped post hoc test is applied. By default, the number of replications is set to 1000. This can be changed into the desired number. 
-        - Effect size: By selecting this option, the effect size (i.e., the magnitude of the observed effect) will be displayed. The used measure for the effect size is Cohen's d. The effect size will only be displayed for the post hoc type `Standard`. 
-
-      -  Games-Howell: This method can be used when equal group/level variances are not assumed. The p-values are corrected with the Tukey method.
-      -  Dunnett: When selecting this method, all the levels are compared to one specific level, for example to the control group. At the moment, it is not possible to manually specify to which level the others levels are compared, but it is based on the order of the levels. To change the order of the levels, the level labels can be adjusted. 
-        <details>
-	        <summary><b>GIF demonstration: Adjust level labels </b></summary>
-	        <img src="analyses/gif/labelediting.gif"/>
-      </details> 
+-  Standard: Pairwise t-tests are performed. All the corrections can be applied to this method. This option is selected by default. 
```

Comment:
Are these changes also present in ancova_nl.md?

Reply:
No it was just a layour error in the english file